test/unit/daemon_tests.rb

```diff
@@ -160,6 +160,12 @@ class InitTests < InitSetupTests
       assert_instance_of configuration.logger.class, data.logger
     end
 
+    should "know its name and pid file" do
+      data = subject.daemon_data
+      assert_equal data.name,     subject.name
+      assert_equal data.pid_file, subject.pid_file
+    end
+
```

Comment:
@jcredding are you missing "should have" regression method tests for these?  And are you missing them for all the imeths?  Not sure but I can't find them.